Formula/sigrok-cli.rb

```diff
@@ -0,0 +1,38 @@
+class SigrokCli < Formula
+  desc "Sigrok command-line interface to use logic analyzers and more"
+  homepage "https://sigrok.org/"
+  url "https://sigrok.org/download/source/sigrok-cli/sigrok-cli-0.7.2.tar.gz"
+  sha256 "71d0443f36897bf565732dec206830dbea0f2789b6601cf10536b286d1140ab8"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url "https://sigrok.org/download/source/sigrok-cli/"
+    regex(/sigrok-cli-(\d+(?:\.\d+)+)\.t/i)
```

Comment:
```suggestion
    url "https://sigrok.org/wiki/Downloads"
    regex(/href=.*?sigrok-cli[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

We generally favor checking a downloads page over a directory listing page, when it's available and kept up-to-date.

As mentioned above, when checking a URL on an HTML page, we also typically restrict matching to `href` attributes and the suggestion above uses the common pattern for this.